FlorenceBML/BMLacq306/BMLacq306.xml

```diff
@@ -338,8 +338,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     <seg type="script">minute and accurate with a more wide pen than <ref target="#h1"/></seg>
```

Comment:
"Minute..." Here and below, what is written inside the mark up element like `<seg type="rubrication"></rubrication>` and the like, stylistically is considered a new sentence, the first word starting with big letter. It has been like this in all descriptions, until now.